tests/conftest.py

```diff
@@ -65,3 +74,60 @@ def image(dev_image):
 def store_path():
     """Path to store zarr images written from benchmarks"""
     return pathlib.Path("data/output/heart-example.zarr")
+
+
+def expand_min_max(config: dict) -> dict:
+    """Expand min/max keys in config to a list of the full range of values."""
+
+    for key in ["chunk_size", "blosc_clevel", "gzip_level", "zstd_level"]:
+        values = config[key]
+        if "min" in values and "max" in values:
+            config[key] = range(values["min"], values["max"] + 1)
+
+    return config
+
+
+def parse_config_files(config_name: str) -> list[dict]:
+    """Parse json config files. 'all' will parse every config in tests/benchmark_configs (except for dev, which contains
+    a small set of parameters for development runs)."""
+
+    configs_dir = pathlib.Path("tests/benchmarks/benchmark_configs")
+    configs = []
+
+    if config_name == "all":
+        for config_file in configs_dir.iterdir():
+            if config_file.is_file() and config_file.stem != "dev":
+                config = expand_min_max(read_json_file(config_file))
+                configs.append(config)
+    else:
+        config_file = configs_dir / f"{config_name}.json"
+        config = expand_min_max(read_json_file(config_file))
+        configs.append(config)
+
+    return configs
+
+
+def pytest_generate_tests(metafunc):
```

Comment:
Could you type this? It would help me (at least) understand what's going on.

Reply:
Will do! I'm mainly following the [`pytest_generate_tests` example from the parametrization docs](https://docs.pytest.org/en/stable/how-to/parametrize.html#basic-pytest-generate-tests-example)